src/compas_occ/brep/brep.py

```diff
@@ -423,30 +443,32 @@ def solids(self) -> List["BRep"]:
 
     @property
     def frame(self) -> compas.geometry.Frame:
-        location = self.occ_shape.Location()
-        transformation = location.Transformation()
-        T = Transformation(
-            matrix=[[transformation.Value(i, j) for j in range(4)] for i in range(4)]
-        )
-        frame = Frame.from_transformation(T)
+        location = self.native_brep.Location()
+        t = location.Transformation()
+
+        # transformation.Value is a 1-based 3x4 matrix
```

Comment:
should this conversion of occ transformation to compas trasnformation be moved into its own? (similar to `compas_rhino.conversions` stuff)?